code/mainmenu.cpp

```diff
@@ -770,8 +776,13 @@ void MainMenu::render()
 			}
 		}
 
-		GUI_RECT rect = { 0, 0, Environment.screenWidth, Environment.screenHeight };
-		drawRect( rect, color );
+		GUI_RECT rect = {

```

Comment:
there is no need to modify it: 
1024 * scaleX = 1024 * (Environment.screenWidth / 1024.0f) = Environment.screenWidth
(same for height)